src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/integration/IntegrationTestBase.kt

```diff
@@ -60,6 +65,15 @@ abstract class IntegrationTestBase {
     stubUser(PROBATION_USER)
   }
 
+  @BeforeEach
+  fun `stub default locations`() {
```

Comment:
Stubbing some default locations here to reduce the duplication across tests.

We could look at doing this for other data e.g. some default prisoners etc.

This should help keep the integration tests a bit cleaner and lighter 🤞